ios/FluentUI/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -5,7 +5,20 @@
 
 import UIKit
 
-class PopupMenuItemCell: TableViewCell {
+class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
+
+    var expectedSeparatorColor: UIColor? {
+        didSet {
+            self.bottomSeparator.backgroundColor = self.expectedSeparatorColor
```

Comment:
>self. [](start = 12, length = 5)

"self." is unnecessary. Can you find all the instances in the new code and changed them?

Reply:
Done.